src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -90,6 +90,17 @@ preflight1 (PreflightContext const& ctx)
         return temBAD_SIGNATURE;
     }
 
+    // An AccountTxnID field constrains transactions more than the Sequence
+    // field.  Tickets, on the other hand, reduce ordering constraints.
+    // A transaction that includes both fields is difficult to get into the
+    // ledger and makes it hard for the TxQ to know the likelihood of a
+    // transactions success.
```

Comment:
I suggest a rephrasing:
```
    // An AccountTxnID field constrains transactions more than the Sequence
    // field.  Tickets, on the other hand, reduce ordering constraints.
    // Because one counteracts the other, the combination is unsupported,
    // and treated a malformed.
```